operator/src/command_utils.rs

```diff
@@ -1,16 +1,22 @@
-use crate::CommandInformation;
-use crate::Error;
 use k8s_openapi::api::core::v1::Pod;
 use k8s_openapi::apimachinery::pkg::apis::meta::v1::Time;
-use k8s_openapi::Metadata;
-use kube::api::{Meta, ObjectMeta};
+use k8s_openapi::Resource;
+use kube::api::{Meta, PostParams};
+use kube::Api;
 use serde::de::DeserializeOwned;
+use serde_json::json;
 use stackable_operator::client::Client;
 use stackable_operator::error::OperatorResult;
-use stackable_spark_crd::commands::{CommandStatus, CommandStatusMessage};
-use stackable_spark_crd::{Restart, Start, Stop};
+use stackable_spark_crd::{ClusterStatus, CurrentCommand, Restart, SparkCluster, Start, Stop};
+use std::collections::HashMap;
 use std::fmt::Debug;
-use tracing::debug;
+
+use stackable_operator::reconcile::ReconcileFunctionAction;
+use std::time::Duration;
+use tracing::info;
+
+const COMMAND_STATUS_LABEL: &str = "status";
```

Comment:
should be something like `spark.stackable.tech/status`

Reply:
Yeah that was lazy ;)